2_process/src/model_diagnostics.R

```diff
@@ -0,0 +1,43 @@
+
+
+## preliminaries 
+library(dplyr)
+library(readr)
+library(stringr)
+library(whisker)
+
+## function that produces model diagnostics
+### default saves output to '2_process/out/model_diagnostic_text.txt
+
+get_diagnostics<-function(input_path = '2_process/out/model_summary_results.csv', output_path = '2_process/out/model_diagnostic_text.txt'){
+  
+  # read in processed data
+  input_data<-readr::read_csv(input_path)
```

Comment:
picky thing, but since we'll all be looking at each other's code a lot, can you put a space between the variable and the assignment. So like this:

```r
input_data <- readr::
```
I see that in a few other spots too.